volestipy/setup.py

```diff
@@ -1,75 +1,83 @@
-from os.path import join
-from setuptools import setup
+# from setuptools import setup
+# from setuptools import Extension
+from distutils.core import setup
+from distutils.core import Extension
 from Cython.Build import cythonize
-from setuptools.extension import Extension
+from os.path import join
 import numpy
 
 
-version = "0.2.0"
+version = "0.3.0"
 license='LGPL3',
 packages = ["volestipy"]
 description="volestipy: wrapper for the VolEsti library to sample from convex sets and compute volume."
-author = "Pedro Zuidberg Dos Martires"
-author_email="pedro.zuidbergdosmartires@cs.kuleuven.be"
+author = "Pedro Zuidberg Dos Martires, Haris Zafeiropoulos"
```

Comment:
you can add your name in credits if you like https://github.com/GeomScale/volume_approximation/blob/develop/doc/credits.md

Reply:
Ok. I did that. :)